dist/index.js.flow

```diff
@@ -0,0 +1,2 @@
+// @flow
+export * from '../src/index.js'
```

Comment:
How about instead of this, the build script explicitly gather the flow types and save them in `dist/index.js.flow` so they're automatically included with the NPM bundle?

Reply:
I was looking for such solution for my libraries but couldn't find one. Flow gen-flow-files, the official solution for it, is completely broken (facebook/flow#5871). Maybe there is something to do within Facebook?